packages/@aws-cdk/aws-autoscaling/test/test.auto-scaling-group.ts

```diff
@@ -149,6 +149,54 @@ export = {
     test.done();
   },
 
+  'can specify only min capacity'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = mockVpc(stack);
+
+    // WHEN
+    new autoscaling.AutoScalingGroup(stack, 'MyFleet', {
+      instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.M4, ec2.InstanceSize.Micro),
+      machineImage: new ec2.AmazonLinuxImage(),
+      vpc,
+      minSize: 10
+    });
+
+    // THEN
+    expect(stack).to(haveResource("AWS::AutoScaling::AutoScalingGroup", {
+        MinSize: "10",
+        MaxSize: "10",
+        DesiredCapacity: "10",
+      }
+    ));
+
+    test.done();
+  },
+
+  'can specify only max capacity'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = mockVpc(stack);
+
+    // WHEN
+    new autoscaling.AutoScalingGroup(stack, 'MyFleet', {
+      instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.M4, ec2.InstanceSize.Micro),
+      machineImage: new ec2.AmazonLinuxImage(),
+      vpc,
+      maxSize: 10
+    });
+
+    // THEN
+    expect(stack).to(haveResource("AWS::AutoScaling::AutoScalingGroup", {
+        MinSize: "10",
```

Comment:
I kind of expected min/desired to be 1 in this case... Not sure

Reply:
My argumentation here is: if you want to have a flexible scale, you'd specify it yourself.

If you say 10, you get exactly 10.

Reply:
OTOH the ApplicationAutoScaling does have default for minimum = 1, so I guess maybe you're right that it's better for consistency.